src/fides/cli/commands/util.py

```diff
@@ -38,69 +39,23 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
     """
 
     executed_at = datetime.now(timezone.utc)
-    separate = lambda: print("-" * 10, end=None)
```

Comment:
I moved all this into the `create_config_file` util function, so I could reuse that logic in `fides deploy` below